core/nginx/dovecot/proxy.conf

```diff
@@ -148,17 +144,12 @@ service lmtp {
 
 service submission-login {
    inet_listener submission {
-{%- if PORT_587 %}
-    port = 587
+    port = {% if PORT_587 %}587{% else %}0{% endif %}
 {%- if PROXY_PROTOCOL_587 %}
     haproxy = yes
-{% endif %}
-{%- else %}
-# if the section is unset the port is bound anyways
-   port = 0
 {% endif %}
    }
-{%- if TLS_465 and PORT_465 %}
```

Comment:
I think it is okay to disallow disabling the necessary ports, but maybe it should throw an error, so people know we do not support disabling the ports by design (and use a firewall ar change the Dockerfile to do so if they really want to).